src/ParseCliServer.js

```diff
@@ -34,8 +34,13 @@ class ParseCliServer {
     parse-cli always send json body but don't send a Content-Type
     header or in some cases send an unexpected value like
     application/octet-stream.
+    express request length limit is very low. Change limit value
+    for fix 'big' files deploy problems.
     */
-    app.use(bodyParser.json({type: function() { return true; }}));
+    let limit = '500mb';
```

Comment:
Faça isso ser configurável. Use `this.config.limit`.

Para isso, você precisa definir o `this.config` no construtor e depois definir `this.config.limit` se já não tiver algum valor.